.github/workflows/release.yml

```diff
@@ -216,6 +155,49 @@ jobs:
           if [ "${{ inputs.dry_run }}" = "true" ]; then  
             box_summary="[Dry run] "
           fi
+            echo "$box_summary :ballot_box: ${{ inputs.release_version }} was uploaded to https://pentaho.app.box.com/folder/$BOX_FOLDER_ID" >> $GITHUB_STEP_SUMMARY
```

Comment:
Please align this `echo` with the above  `if`.

Reply:
changed

Reply:
I still see it not aligned (posting image):
![image](https://github.com/pentaho/actions-common/assets/13484152/e92aefd6-6077-4da0-a14f-28586a6bec5e)